### DISCUSSION TITLE
Hashicorp logos

### DISCUSSION THREAD

**Discussion by adamantike** [selfhst/icons#44] [New Icon Request]
Official website: https://www.hashicorp.com/

Product logos: https://brand.hashicorp.com/product_logos

Products:
* Terraform
* Vagrant
* Packer
* Waypoint
* Nomad
* Vault
* Boundary
* Consul

Each of their products include both the Product logo, and variations for the Cloud, Self-hosted, and Community variants, as explained in the linked website.

**Comment by shollyethan** [comment]
@adamantike Are you interested in having me add all the variations to the collection, or fine with just the primary given they're all so similar?

Not opposed to either, just checking before I go through the effort.

**Comment by shollyethan** [comment]
I've added the 'Product' logo for each of the products to the collection. Note that I've used the product names as the icon name for each but have prefaced the reference for each with 'hashicorp-' for easier searching/discovery.

I'm happy to add the other variations in the future if anyone is interested.

**Comment by adamantike** [comment]
I think that's enough for now, thanks for the quick addition!
